### PR TITLE
13 dt cli provide more helpful error message on yaml

### LINKED ISSUE #13: Provide more helpful error message on yaml parsing problems
Currently in case there's a problem with parsing `extension.yaml` file user is presented with a stack trace without any hint what's exactly going on. While I don't have a problem with the stacktrace, I believe we should also display a message stating where the problem comes from. Ideally, we should distinguish syntax/formatting problems from missing required fields or unexpected/wrong values.

Ideally, we should report offending line numbers and character positions usually available in [YAMLError](https://pyyaml.org/wiki/PyYAMLDocumentation#yamlerror).

### PR DESCRIPTION
**Pull request by TDyszewski**
Fixes #13

### CODE REVIEW

**Comment by dyladan** [comment]
[CLA assistant check](https://cla.oss.dynalabs.io/dynatrace-oss/dt-cli?pullRequest=28) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.oss.dynalabs.io/dynatrace-oss/dt-cli?pullRequest=28) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.oss.dynalabs.io/check/dynatrace-oss/dt-cli?pullRequest=28) it.</sub>

**Review by wbachnik — APPROVED** [review]

File: dtcli/building.py [hunk 1]
```diff
@@ -70,7 +70,12 @@ def _package(
 ):
     extension_yaml_path = os.path.join(extension_dir_path, EXTENSION_YAML)
     with open(extension_yaml_path, "r") as fp:
-        metadata = yaml.safe_load(fp)
+        try:
+            metadata = yaml.safe_load(fp)
+        except Exception as e:
```
Comment (allgreed) [inline dtcli/building.py:75]: Can we target a sepcific yaml exception? What does it throw? ValueError?

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # IDE
+.idea/
```
Comment (allgreed) [inline .gitignore:2]: I think this should be handled at individual level, by a global gitignore

**Comment by allgreed** [comment]
We're working with @Dysio on exactly this feature, but it'll happen in a separate PR, because for reasons™

**Pull request closed without merging**